run_sdk_container

```diff
@@ -104,6 +108,9 @@ if [ -z "$stat" ] ; then
 
     gpg_volumes=$(gnupg_ssh_gcloud_mount_opts)
 
+    source ci-automation/ci_automation_common.sh
+    docker_image_from_registry_or_buildcache "flatcar-sdk-${arch}" "${docker_sdk_vernum}"
```

Comment:
@pothos unfortunately this breaks when a custom container name is supplied. `${docker_sdk_vernum}` is never calculated in that case since it was previously only used cosmetically by `run_sdk_container`, to generate the container name. Since all of the `ci-automation` scripts use a custom container name, all the nightlies just broke :sweat_smile: (No worries, arguably, that's what they are for).
Could you please have a look?

Reply:
https://github.com/flatcar-linux/scripts/pull/260

Reply:
Ah, sorry, forgot to test with the new pipeline…